### PR TITLE
Exclude dependency-based query from C# Code Scanning

### PR DESCRIPTION
**Pull request by rneatherway**
This query overlaps with tools such as dependabot.

### CODE REVIEW

**Comment by rneatherway** [comment]
@robertbrignull I noticed there was a typo in one of the cpp suites, which I've also corrected here. How would you recommend testing these changes?

**Comment by robertbrignull** [comment]
One way you could test this change is to checkout this commit locally and run `codeql resolve queries cpp-security-extended.qls` and it should work but then fail if you reintroduce the typo.

File: cpp/ql/src/codeql-suites/cpp-security-extended.qls [hunk 1]
```diff
@@ -2,5 +2,5 @@
 - qlpack: codeql-cpp
 - apply: security-extended-selectors.yml
   from: codeql-suite-helpers
-- apply: codeql-suites/excluded-slow-queries.yml
+- apply: codeql-suites/exclude-slow-queries.yml
```
Comment (jbj) [inline cpp/ql/src/codeql-suites/cpp-security-extended.qls:5]: This has been fixed already on `master`, but repeating the fix shouldn't hurt.

**Comment by rneatherway** [comment]
Please go ahead and merge if you're happy, I don't have write access here.

**Pull request merged**